incuna-sass/helpers/_link.sass

```diff
@@ -1,18 +1,19 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
-        text-decoration: underline
-        &:hover
-            text-decoration: none
     @if $type = reverse
         text-decoration: none
         &:hover
             text-decoration: underline
-    @if $type = blank
-        text-decoration: none
-        &:hover
+    @else
```

Comment:
In this example the gain is negligible, but in general it is much easier to follow the logic. Things like this can quickly grow into spagetti, where you don't know anymore which level are you in. It works the same, it's only for maintainance and ease of use.